Sources/Foundation.swift

```diff
@@ -143,13 +143,13 @@ extension JSON {
         public var ns: Any {
             switch self {
             case .null: return NSNull()
-            case .bool(let b): return NSNumber(value: b)
-            case .string(let s): return s
-            case .int64(let i): return NSNumber(value: i)
-            case .double(let d): return d
-            case .decimal(let d): return NSDecimalNumber(decimal: d)
-            case .object(let obj): return obj.ns
-            case .array(let ary):
+            case let .bool(b): return NSNumber(value: b)
+            case let .string(s): return s
+            case let .int64(i): return NSNumber(value: i)
+            case let .double(d): return d
+            case let .decimal(d): return NSDecimalNumber(decimal: d)
+            case let .object(obj): return obj.ns
+            case let .array(ary):
```

Comment:
My personal style is to keep the `let`s as close as possible to the thing being bound, as long as that doesn't introduce multiple `let`s.

Examples:

```swift
swift foo {
case (_, let a): …
case let (a, b): …
case (_, let (a, b)): …
case (let a, (someValueToMatch, let b)): …
}
```